### PR TITLE
[CUBRIDQA-1117] Remove system parameter 'dont_reuse_heap_file' in testcases.

### PR DESCRIPTION
**Pull request by tw-kang**
refer to : http://jira.cubrid.org/browse/CBRD-23866
http://jira.cubrid.org/browse/CUBRIDQA-1117

### CODE REVIEW

File: sql/_01_object/_02_class/_006_reusable_oid/cases/_016_insert_index.sql [hunk 1]
```diff
@@ -87,5 +86,4 @@ insert into foo select a+100, b+100 from foo order by a+10;
 select /*+ USE_IDX */ * from foo where a > 0 order by 1;
 
 drop table foo;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_01_object/_02_class/_006_reusable_oid/cases/_016_insert_index.sql:90]: need to save 'commit;'

File: sql/_07_misc/_01_statistics/cases/CUBRIDSUS853.sql [hunk 2]
```diff
@@ -125,5 +124,4 @@ from foo
 where i < 0 and i > 10
 
 drop foo;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_07_misc/_01_statistics/cases/CUBRIDSUS853.sql:128]: need to save 'commit;'

File: sql/_13_issues/_10_1h/cases/bug_4126.sql [hunk 3]
```diff
@@ -73,5 +72,4 @@ insert into z values(14,14,14,14);
 update statistics on z;
 select /*+ recompile */ * from z where a = 5 and b = 5;
 drop table z;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_13_issues/_10_1h/cases/bug_4126.sql:76]: need to save 'commit;'

File: sql/_13_issues/_10_2h/cases/bug_4278.sql [hunk 4]
```diff
@@ -23,5 +22,4 @@ using index i_foo_a_b_c_d_d_d
 order by c desc, d desc limit 1;
 
 drop table foo;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_13_issues/_10_2h/cases/bug_4278.sql:26]: need to save 'commit;'

File: sql/_14_mysql_compatibility_2/_12_type_conversions/_11_number_comparisons/cases/1002.sql [hunk 5]
```diff
@@ -163,5 +162,4 @@ select * from t where i = '2' order by i;
 select * from t where i between 1 and 4 order by i;
 select * from t where i in (1, 2, 3) order by i;
 drop table t;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_14_mysql_compatibility_2/_12_type_conversions/_11_number_comparisons/cases/1002.sql:166]: need to save 'commit;'

File: sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_02_index_covering.sql [hunk 6]
```diff
@@ -45,5 +44,4 @@ drop table tbl;
 drop table d1;
 drop table d2;
 drop table d3;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_02_index_covering.sql:48]: need to save 'commit;'

File: sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_03_index_covering.sql [hunk 7]
```diff
@@ -56,5 +55,4 @@ drop table tbl;
 drop table d1;
 drop table d2;
 drop table d3;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_03_index_covering.sql:59]: need to save 'commit;'

File: sql/_16_index_enhancement/_07_index_keylimit_plan_dump/cases/_03_index_keylimit.sql [hunk 8]
```diff
@@ -4058,5 +4057,4 @@ drop table tab2;
 drop table tab3;
 drop table test_table;
 drop table t1;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_16_index_enhancement/_07_index_keylimit_plan_dump/cases/_03_index_keylimit.sql:4061]: need to save 'commit;'

File: sql/_16_index_enhancement/_12_descending_index_scan/cases/_03_order_by_skip(dump).sql [hunk 9]
```diff
@@ -49,5 +48,4 @@ SELECT j, i, s FROM t WHERE j > 0 ORDER BY j DESC, i;
 SELECT /*+ use_desc_idx */ * FROM t WHERE i > 0;
 
 DROP TABLE t;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_16_index_enhancement/_12_descending_index_scan/cases/_03_order_by_skip(dump).sql:52]: need to save 'commit;'

File: sql/_16_index_enhancement/_12_descending_index_scan/cases/_09_order_by_skip_2(dump).sql [hunk 10]
```diff
@@ -45,5 +44,4 @@ SELECT * FROM t WHERE k = 0 and j > 0 ORDER BY j desc;
 SELECT * FROM t WHERE i > 0 AND j = 63586 ORDER BY i desc, j, k, s;
 
 DROP TABLE t;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_16_index_enhancement/_12_descending_index_scan/cases/_09_order_by_skip_2(dump).sql:48]: need to save 'commit;'

File: sql/_16_index_enhancement/_13_index_grouping/cases/_011_index_grouping_11.sql [hunk 11]
```diff
@@ -56,5 +55,4 @@ select sum(a*b*c*d*e) from xoo3 group by a, b, c, d;
 select sum(a+b+c), e from xoo3 group by b, c, d;
 
 drop class xoo3;
-set  system parameters 'dont_reuse_heap_file=no';commit;
```
Comment (tw-kang) [inline sql/_16_index_enhancement/_13_index_grouping/cases/_011_index_grouping_11.sql:59]: need to save 'commit;'

File: sql/_01_object/_02_class/_006_reusable_oid/cases/_016_insert_index.sql (hunk 1 above)
Comment (tw-kang) [inline sql/_01_object/_02_class/_006_reusable_oid/cases/_016_insert_index.sql:90]: need to save 'commit;'

File: sql/_07_misc/_01_statistics/cases/CUBRIDSUS853.sql (hunk 2 above)
Comment (tw-kang) [inline sql/_07_misc/_01_statistics/cases/CUBRIDSUS853.sql:128]: need to save 'commit;'

File: sql/_13_issues/_10_1h/cases/bug_4126.sql (hunk 3 above)
Comment (tw-kang) [inline sql/_13_issues/_10_1h/cases/bug_4126.sql:76]: need to save 'commit;'

File: sql/_13_issues/_10_2h/cases/bug_4278.sql (hunk 4 above)
Comment (tw-kang) [inline sql/_13_issues/_10_2h/cases/bug_4278.sql:26]: need to save 'commit;'

File: sql/_14_mysql_compatibility_2/_12_type_conversions/_11_number_comparisons/cases/1002.sql (hunk 5 above)
Comment (tw-kang) [inline sql/_14_mysql_compatibility_2/_12_type_conversions/_11_number_comparisons/cases/1002.sql:166]: need to save 'commit;'

File: sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_02_index_covering.sql (hunk 6 above)
Comment (tw-kang) [inline sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_02_index_covering.sql:48]: need to save 'commit;'

File: sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_03_index_covering.sql (hunk 7 above)
Comment (tw-kang) [inline sql/_16_index_enhancement/_05_index_covering_plan_dump/cases/_03_index_covering.sql:59]: need to save 'commit;'

File: sql/_16_index_enhancement/_07_index_keylimit_plan_dump/cases/_03_index_keylimit.sql (hunk 8 above)
Comment (tw-kang) [inline sql/_16_index_enhancement/_07_index_keylimit_plan_dump/cases/_03_index_keylimit.sql:4061]: need to save 'commit;'

File: sql/_16_index_enhancement/_12_descending_index_scan/cases/_03_order_by_skip(dump).sql (hunk 9 above)
Comment (tw-kang) [inline sql/_16_index_enhancement/_12_descending_index_scan/cases/_03_order_by_skip(dump).sql:52]: need to save 'commit;'

File: sql/_16_index_enhancement/_12_descending_index_scan/cases/_09_order_by_skip_2(dump).sql (hunk 10 above)
Comment (tw-kang) [inline sql/_16_index_enhancement/_12_descending_index_scan/cases/_09_order_by_skip_2(dump).sql:48]: need to save 'commit;'

File: sql/_16_index_enhancement/_13_index_grouping/cases/_011_index_grouping_11.sql (hunk 11 above)
Comment (tw-kang) [inline sql/_16_index_enhancement/_13_index_grouping/cases/_011_index_grouping_11.sql:59]: need to save 'commit;'

**Review by tw-kang — COMMENTED** [review]
need to save 'commit;'
caused by the removed 'commit' drop table  did not executed well.

File: sql/_18_index_enhancement_qa/_01_acceptance_test/cases/_04_like_01.sql [hunk 12]
```diff
@@ -11,6 +10,5 @@ SELECT /*+ RECOMPILE */ a,b FROM t1 WHERE b LIKE '%testing 3' order by a;
 
 SELECT /*+ RECOMPILE */ a,b FROM t1 WHERE b LIKE '%test%' order by a;
 
-DROP TABLE t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_01_acceptance_test/cases/_04_like_01.sql:14]: need to save

File: sql/_18_index_enhancement_qa/_01_acceptance_test/cases/_06_limit_optimation_01.sql [hunk 13]
```diff
@@ -18,6 +17,5 @@ SELECT /*+ RECOMPILE */ a,b FROM t1 WHERE a>0 ORDER BY a LIMIT 5;
 --TEST: rownum rewrite could not be adopt, but require TOP N sorting
 SELECT /*+ RECOMPILE */ a,b FROM t1 WHERE b=1 ORDER BY a LIMIT 5;
 
-DROP TABLE t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_01_acceptance_test/cases/_06_limit_optimation_01.sql:21]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_05_data_type_basic_subquery.sql [hunk 14]
```diff
@@ -90,6 +89,5 @@ select /*+ RECOMPILE */ count(b),10 from t1 where exists (select t1.a) ;
 --TEST: should use i_t1_a2a
 select /*+ RECOMPILE */ count(length(a)),10 from t1 where exists (select t1.a) ;
 
-drop table t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_05_data_type_basic_subquery.sql:93]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_07_data_type_basic_createtable_like.sql [hunk 15]
```diff
@@ -53,6 +52,5 @@ select /*+ RECOMPILE */ g from t2 where g > -1;
 select /*+ RECOMPILE */ q from t2 where q > TIMESTAMP '2010-10-31 01:15:50';
 
 drop table t1;
-drop table t2;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_07_data_type_basic_createtable_like.sql:56]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_09_data_type_basic_prepare_statment.sql [hunk 16]
```diff
@@ -126,6 +125,5 @@ PREPARE stmt1 FROM 'select /*+ RECOMPILE */ a from t1'
 execute stmt1
 DEALLOCATE PREPARE stmt1;
 
-drop table t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_09_data_type_basic_prepare_statment.sql:129]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_10_data_type_basic_prepare_statment.sql [hunk 17]
```diff
@@ -126,6 +125,5 @@ PREPARE stmt1 FROM 'select /*+ RECOMPILE */ q from t1'
 execute stmt1
 DEALLOCATE PREPARE stmt1;
 
-drop table t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_10_data_type_basic_prepare_statment.sql:129]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_11_data_type_basic_truncate.sql [hunk 18]
```diff
@@ -127,6 +126,5 @@ PREPARE stmt1 FROM 'select /*+ RECOMPILE */ q from t1'
 execute stmt1
 DEALLOCATE PREPARE stmt1;
 
-drop table t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_11_data_type_basic_truncate.sql:130]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_12_data_type_basic_delete.sql [hunk 19]
```diff
@@ -127,6 +126,5 @@ PREPARE stmt1 FROM 'select /*+ RECOMPILE */ q from t1'
 execute stmt1
 DEALLOCATE PREPARE stmt1;
 
-drop table t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_01_covering_index/cases/_t110_12_data_type_basic_delete.sql:130]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_04_index_like/cases/_t4_08_underscore.sql [hunk 20]
```diff
@@ -27,6 +26,5 @@ select /*+ recompile */ * from t1 where v like '____%';
 
 
 
-drop table t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_04_index_like/cases/_t4_08_underscore.sql:30]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_04_index_like/cases/_t4_09_escape.sql [hunk 21]
```diff
@@ -18,6 +17,5 @@ insert into t1 values (9, null, null, null);
 
 select /*+ recompile */ * from t1 where v like '\%%' escape '\';
 
-drop table t1;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_04_index_like/cases/_t4_09_escape.sql:21]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_06_optimizing_limit/cases/_03_composite_index.sql [hunk 22]
```diff
@@ -16,6 +15,5 @@ select /*+ recompile */ * from t where i1 between 1 and 3 and i3 between '00' an
 --TEST
 select /*+ recompile */ * from t where i1 between 1 and 3 and i2 between 2 and 4 and i3 between '00' and '44' limit 6;
 
-drop table t;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_06_optimizing_limit/cases/_03_composite_index.sql:19]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_06_optimizing_limit/cases/_04_orderby_num_2_rownum.sql [hunk 23]
```diff
@@ -19,6 +18,5 @@ select * from (select /*+ recompile */ * from t where i1 > 0 order by i1 desc fo
 select * from (select /*+ recompile */ * from t where i1 > 0 order by i1, i2, i3 desc for orderby_num() between 3 and 10) t order by 1,2,3;
 
 
-drop table t;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_06_optimizing_limit/cases/_04_orderby_num_2_rownum.sql:22]: need to save

File: sql/_18_index_enhancement_qa/_02_full_test/_08_ordering_index/cases/_t120_04_ordering_index_similiar.sql [hunk 24]
```diff
@@ -59,6 +58,5 @@ select /*+ recompile */ b,c,d from t where b=1 and c>1 and d>0 order by c;
 --should use i_t_bdc
 select /*+ recompile */ b,c,d from t where b=1 and c>1 and d>0 order by d;
 
-drop table t;set system parameters 'dont_reuse_heap_file=no';
```
Comment (tw-kang) [inline sql/_18_index_enhancement_qa/_02_full_test/_08_ordering_index/cases/_t120_04_ordering_index_similiar.sql:62]: need to save

**Review by tw-kang — COMMENTED** [review]
caused by 'drop table ~' is removed

**Review by ssihil — APPROVED** [review]

**Pull request merged**